core/renderers/zelos/info.js

```diff
@@ -199,6 +180,20 @@ Blockly.zelos.RenderInfo.prototype.getSpacerRowHeight_ = function(
   return this.constants_.MEDIUM_PADDING;
 };
 
+/**
+ * @override
+ */
+Blockly.zelos.RenderInfo.prototype.getSpacerRowWidth_ = function(prev, next) {
+  var width = this.width - this.startX;
+  if (Blockly.blockRendering.Types.isInputRow(prev) && prev.hasStatement) {
+    return Math.max(width, this.constants_.STATEMENT_INPUT_SPACER_MIN_WIDTH);
+  }
+  if (Blockly.blockRendering.Types.isInputRow(next) && next.hasStatement) {
+    return Math.max(width, this.constants_.STATEMENT_INPUT_SPACER_MIN_WIDTH);
```

Comment:
nit: These could be combined since they are returning the same thing.

Reply:
Done.